### ISSUE TITLE
[java] 3 rules reporting a comment twice for same line

### ISSUE THREAD

**Issue by dinesh150** [pmd/pmd#3461]
**Affects PMD Version: 6.37.0** 
**Description: Three rules from java reporting a comment twice for same line**

DoNotTerminateVM, ImplicitSwitchFallThrough and ComparisonWithNaN

These three rules from java produces report twice for same line.

**Code Sample demonstrating the issue:**

```java
// Code is sample for ComparisonWithNaN rule.
 public class Sample{
     public void doSomething(){
        double test = 0;
	boolean rule = (test == Double.NaN);
    }
}
```

**Steps to reproduce:**
        Use the shared code and shared rule name from CLI.

**Running PMD through:** *[CLI]*

**Comment by adangel** [comment]
Can you share your ruleset?

All three rules have been renamed lately:
* ComparisonWithNaN was previously called BadComparison
* ImplicitSwitchFallThrough was previously called MissingBreakInSwitch
* DoNotTerminateVM was previously called DoNotCallSystemExit

The old rule names are still available, but deprecated: https://github.com/pmd/pmd/blob/efaf34df44e059d3953b1aaec8008672c46812d9/pmd-java/src/main/resources/category/java/errorprone.xml#L641

If you use the complete errorprone category, the old (deprecated) rules should be ignored. You should see duplicated reports only if you explicitly enable e.g. BadComparison and ComparisonWithNaN (which are essentially the same rule).

**Comment by jsotuyod** [comment]
Marking this as stale after 3 years with no response from the reporter nor a way to reproduce it.
